assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -192,8 +193,30 @@ void EntityTreeSendThread::startNewTraversal(const ViewFrustum& view, EntityTree
                 AACube cube = entity->getQueryAACube(success);
                 if (success) {
                     if (_traversal.getCurrentView().cubeIntersectsKeyhole(cube)) {
-                        float priority = _conicalView.computePriority(cube);
-                        _sendQueue.push(PrioritizedEntity(entity, priority));
+                        // Check the size of the entity, it's possible that a "too small to see" entity is included in a
+                        // larger octree cell because of its position (for example if it crosses the boundary of a cell it
+                        // pops to the next higher cell. So we want to check to see that the entity is large enough to be seen
+                        // before we consider including it.
+                        // We can't cull a parent-entity by its dimensions because the child may be larger.  We need to
+                        // avoid sending details about a child but not the parent.  The parent's queryAACube should have
+                        // been adjusted to encompass the queryAACube of the child.
+                        AABox entityBounds = entity->hasChildren() ? AABox(cube) : entity->getAABox(success);
+                        if (!success) {
+                            // if this entity is a child of an avatar, the entity-server wont be able to determine its
+                            // AABox.  If this happens, fall back to the queryAACube.
+                            entityBounds = AABox(cube);
+                        }
+
+                        float renderAccuracy = calculateRenderAccuracy(_traversal.getCurrentView().getPosition(),
+                                                                       entityBounds,
+                                                                       octreeSizeScale,
+                                                                       lodLevelOffset);
```

Comment:
Getting the bounding box a second time bums me out.  I'd like to keep the traversal+collection to be as fast as possible.  To that end, let's just use the existing bounding `cube` for the `calculateRenderAccuracy()` method.  It is an approximation but will be good enough, and we won't have to bother checking whether the entity has children or not.  If necessary let's implement a version of `calculateRenderAccuracy()` that takes an `AACube` rather than an `AABox`.